src/transformers/tokenization_utils_base.py

```diff
@@ -576,7 +576,7 @@ def convert_to_tensors(
         return self
 
     @torch_required
-    def to(self, device: str) -> "BatchEncoding":
+    def to(self, device: Union[str, torch.device]) -> "BatchEncoding":
```

Comment:
This will fail when `torch` is not installed. Can you put it between `"`?
```suggestion
    def to(self, device: Union[str, "torch.device"]) -> "BatchEncoding":
```